docs/api/paths/admin/keyspace-_raw-docid.yaml

```diff
@@ -43,7 +43,7 @@ get:
             additionalProperties: true
             properties:
               _xattrs:
-                description: The extended attributes of the document, which can include Sync Gateway metadata, such as revision history, channel information, and other sync-related data. This is non-exhaustive and only includes xattrs that are relevant to Sync Gateway.
+                description: The extended attributes of the document, which can include Sync Gateway metadata, such as revision history, channel information, and other sync-related data. This is non-exhaustive and only includes xattrs that are relevant to Sync Gateway. Any `null` values indicate the xattr is not present on the document.
```

Comment:
The way in openapi to do this is in OpenAPI 3.0 (currently set):

`nullable: true` https://swagger.io/docs/specification/v3_0/data-models/data-types/#null

At a soon but later point, we can update to 3.1 (not a big deal, mostly just changes null types), and you can do `type: [object, null]`